go/src/lib.rs

```diff
@@ -512,7 +515,8 @@ fn valkey_value_to_command_response(value: Value) -> RedisResult<CommandResponse
 ///
 /// # Safety
 ///
-/// * TODO: finish safety section.
+/// This function should only be called, and should complete and call one of the response callbacks before `close_client` is called.
```

Comment:
Same here. I don't know what "should only be called" means. Is there something else that can be done with the function?

Reply:
you're right, the doc was missing "once".